app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt

```diff
@@ -78,7 +78,7 @@ fun KeyboardKey(
     capsLock: Boolean,
     legendSize: Int,
     keyPadding: Int,
-    keyBorderWidth: Int,
+    keyBorderWidth: Float,
```

Comment:
Why does this need to change to a float, its an integer in the DB. You could just convert it to a float where necessary.

Reply:
Right you are, not quite sure what I was doing.

Reply:
No probs!